.circleci/config.yml

```diff
@@ -9,13 +9,13 @@ orbs:
 executors:
   nodejs-browsers:
     docker:
-      - image: votingworks/cimg-debian11-browsers:2.0.4
+      - image: votingworks/cimg-debian11-browsers:2.0.5
```

Comment:
Do we need to update vxsuite-build-system at all to include `libzbar-dev`, e.g. maybe in this file? https://github.com/votingworks/vxsuite-build-system/blob/main/playbooks/install-vxsuite-packages.yaml

Reply:
Unsure. I created https://github.com/votingworks/vxsuite-build-system/pull/85 just in case.